libp2p/protocols/pubsub/pubsubpeer.nim

```diff
@@ -288,22 +288,25 @@ proc sendMsgSlow(p: PubSubPeer, msg: seq[byte]) {.async.} =
   if p.sendConn == nil:
     # Wait for a send conn to be setup. `connectOnce` will
     # complete this even if the sendConn setup failed
-    await p.connectedFut
+    try:
+      await p.connectedFut
+    except CatchableError as exc:
+      debug "Unable to get a send connection", msg = exc.msg, p
 
   var conn = p.sendConn
   if conn == nil or conn.closed():
-    debug "No send connection", p, msg = shortLog(msg)
+    debug "No send connection", msg = shortLog(msg), p
     return
 
-  trace "sending encoded msg to peer", conn, encoded = shortLog(msg)
+  trace "sending encoded msg to peer", conn, encoded = shortLog(msg), p
   await sendMsgContinue(conn, conn.writeLp(msg))
```

Comment:
This place could also be source of `CancelledError`.
Every `await` could be source.

Reply:
even if `sendMsgContinue` doesn't raise any error?

Reply:
of course.